javascript-forms/form-events.js

```diff
@@ -0,0 +1,29 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('event.target.name and value:', event.target.name + ':' + event.target.value);
+}
+
+const $userName = document.forms[0].name;
+const $userEmail = document.forms[0].email;
+const $userMessage = document.forms[0].message;
```

Comment:
The problem with getting a form by its index within the `document` is that it'll break if any other `<form>`s are added to the page before it. Try selecting it by something that isn't dependent on the HTML's source order.